.github/workflows/reusable-labelsync.yml

```diff
@@ -0,0 +1,30 @@
+name: label-sync
+
+on:
+  workflow_call:
+    inputs:
+      operating-system:
+        description: The operating system to use (default `ubuntu-latest`)
+        required: false
+        default: ubuntu-latest
+        type: string
+
+jobs:
+  label-sync:
+    name: Label sync
+    runs-on: ${{ inputs.operating-system }}
```

Comment:
Do we really need this input from the workflow caller?

Reply:
Not necessarily no, just makes it possible to choose another OS. Do you want me to change it to ubuntu-latest?

Reply:
@Mino03947 ?

Reply:
Yes. I don't see a need for it in the future.